onebusaway-android/build.gradle

```diff
@@ -299,11 +299,13 @@ tasks.whenTaskAdded { theTask ->
 
 dependencies {
     // Google Play Services Analytics (we need this on Amazon flavor too)
-    compile('com.google.android.gms:play-services-analytics:9.0.1') {
+    compile('com.google.android.gms:play-services-analytics:9.2.0') {
         exclude module: 'play-services-ads' // See #342
     }
     // Google Play Services Location (we need this on Amazon flavor too)
-    compile 'com.google.android.gms:play-services-location:9.0.1'
+    compile 'com.google.android.gms:play-services-location:9.2.0'
+    // Google Play Services Places library is required by ProprietaryMapHelpV2
```

Comment:
Sorry I missed this in the initial review - if `compile 'com.google.android.gms:play-services-places:9.2.0'` is only needed when building the google build variants due to use in `ProprietaryMapHelpV2`, then this line should change to `googleCompile 'com.google.android.gms:play-services-places:9.2.0'` and should be moved below under the `// Google Play Services Maps (only for Google flavor)`.  This will prevent the library from being unnecessarily compiled into the amazon variants.